classes/class-send-products.php

```diff
@@ -105,6 +104,71 @@ public static function get_product_type_price($product)
             );
         }
 
+
+        /**
+         * Checks if stock management is enabled and if so, returns quantity and status
+         *
+         * @param  mixed $product Product to check   
+         * @return array ['stock_quantity' => $stock_quantity,'stock_status' => $stock_status] Array with quantity and status. if stock management is disabled, false will be returned,
+         */
+        public static function get_product_stock_data($product)
+        {
+            if ($product->get_manage_stock()) {
+                return array(
+                    'stock_quantity' => $product->get_stock_quantity(),
+                    'stock_status' => $product->get_stock_status()
+                );
+            } else {
+                return false;
+            }
+        }
+
+        /**
+         * Get attributes from product
+         *
+         * @param  mixed $product Product to check   
+         * @return array ['pa_name' => ['value1', 'value2']] Array with key set to the product attribute internal name and values as array. returns false if not attributes found.
+         */
+        public static function get_product_attributes($product)
+        {
+            $rawAttributes = $product->get_attributes();
+            $numericRangeAttributes = ["pa_height", "pa_flowermonth"];
```

Comment:
@SublimeProphets 

Why are the table names hardcoded?

Reply:
As mentioned above this was not intended. I will close this PR and open a new one with plenty of added features :)